app/components/pair-compare.jsx

```diff
@@ -268,11 +285,19 @@ const useStyles = createUseStyles(theme => ({
     transform: 'translateX(200%)',
     transition: 'transform 0.5s linear',
   },
+  customButton: {
+    width: '25rem',
+    '@media (max-width: 600px)': {
+      width: '75%',
+      maxWidth: '17rem',
+    },
+  },
 }))
 
 const sharedStatusBadgeStyle = () => ({
   borderRadius: '1rem',
   padding: '0.375rem 0.625rem',
+  marginBottom: '20rem',
```

Comment:
something seems wrong with a margin this big, and when I tried changing it I didn't see a difference. Can you take it out and see if it still works okay, and if it's needed can you point me to the test case.